api/src/domain-logic/en-vi.js

```diff
@@ -27,3 +27,25 @@ where word = ?`
     })),
   }
 }
+
+exports.count = async function count() {
+  const sql = 'SELECT COUNT(DISTINCT word) as count FROM words'
+
+  const summary = db.prepare(sql).get()
+
+  return summary.count
+}
+
+exports.getSuggestions = async function ({ word, limit = 10 }) {
+  let limitNumber = parseInt(limit)
+  limitNumber = limitNumber > 50 || limitNumber < 0 ? 50 : limitNumber
```

Comment:
or `limitNumber = Math.max(0, Math.min(limitNumber, 50))`. Either way works, yours may be easier to read and maintain.